proxy/http/HttpSM.cc

```diff
@@ -1907,6 +1907,12 @@ HttpSM::state_read_server_response_header(int event, void *data)
   case VC_EVENT_EOS:
     server_entry->eos = true;
 
+    if (vio->ndone == 0) {
```

Comment:
This part is probably correct, but there is a similar code below.
https://github.com/apache/trafficserver/blob/e8bb4909bc2c948250ba860a71b8a08f853a4275/proxy/http/HttpSM.cc#L1993-L1994

How about make this parse error and handle it later on the line?

Reply:
@shinrich removed these code by bba6321e6305f6321ebb5d571c5ad382ab3acf20.